src/experiments.jl

```diff
@@ -58,8 +58,11 @@ function check_params(savepath, data, parameters)
 	fs = filter(x->!(startswith(x, "model")), fs)
 	# if the first argument name contains a "_", than the savename is parsed wrongly
 	saved_params = map(x -> DrWatson.parse_savename("_"*x)[2], fs)
+	# now filter out saved models where parameter names are different or missing
+	pkeys = collect(keys(parameters))
+	filter!(ps->intersect(pkeys, Symbol.(collect(keys(ps))))==pkeys, saved_params)
 	for params in saved_params
-		all(map(k->params[String(k)] == parameters[k], collect(keys(parameters)))) ? (return false) : nothing
```

Comment:
I've had in mind something, that will ignore skip those files that do not match the same key names. This is however trying to match the values on the intersection of names, which I don't really consider as match as they may come from some weird version of the model.

Oh I see now, that what if the intersection is not equal to pkeys, it will skip the for cycle and go straight to the end. Leaving the above just as a note, that the code is not as clear even for someone who wrote the issue.

Reply:
Yes, the added line is supposed to filter out models where some of the new parameter names are missing.